src/main/java/graphql/schema/DataFetchingEnvironment.java

```diff
@@ -49,11 +46,12 @@
      * is invoked
      *
      * @param <T> you decide what type it is
-     *
      * @return a context object
      */
     <T> T getContext();
 
```

Comment:
We should add JavaDoc here for consistency AND this is also a key difference to getSource()

- [ ] Add JavaDoc